lib/go-atscfg/meta.go

```diff
@@ -356,29 +359,32 @@ func filterConfigFileDSes(server *Server, deliveryServices []DeliveryService, de
 				warnings = append(warnings, "got deliveryservice with nil id, skipping!")
 				continue
 			}
-			if ds.XMLID == nil {
+			if &ds.XMLID == nil {
```

Comment:
This condition should be `ds.XMLID == ""`

Reply:
done